### PR TITLE
[KOGITO-1861] - Make possible to turn the Maven downloading process o…

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
…utput off during s2i builds

Jira issue : https://issues.redhat.com/browse/KOGITO-1861

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a testcase that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Review by sutaakar — APPROVED** [review]

**Review by spolti — APPROVED** [review]

**Comment by spolti** [comment]
I forget one thing, can you please update the README with this new env?

**Comment by vaibhavjainwiz** [comment]
> I forget one thing, can you please update the README with this new env?

@spolti I had updated readme file

**Review by spolti — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -311,7 +311,7 @@ sys	0m0.539s
 ```
 
 Here you can see that the build time has again been reduced. 
-If the maven mirror have already have all the dependencies there, the build time can be even faster.
+If the maven mirror have already have all the dependencies there, the build time can be even faster. Maven generates lots of transfer logs for downloading/uploading of maven dependencies. By default, these logs are disabled. To view these logs we need to set env variable **MAVEN_DOWNLOAD_OUTPUT** to true.
```
Comment (radtriste) [inline README.md]:
```suggestion
If the maven mirror have already have all the dependencies there, the build time can be even faster.  
Also, Maven generates lots of transfer logs for downloading/uploading of maven dependencies. By default, these logs are disabled. To view these logs we need to set env variable **MAVEN_DOWNLOAD_OUTPUT** to true. 
```

**Review by radtriste — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (spolti) [reply README.md]: @vaibhavjainwiz can you please applu @radtriste's suggestions?

Comment (vaibhavjainwiz) [reply README.md]: > @vaibhavjainwiz can you please applu @radtriste's suggestions?

@spolti done

File: README.md [hunk 2]
```diff
@@ -311,7 +311,8 @@ sys	0m0.539s
 ```
 
 Here you can see that the build time has again been reduced. 
-If the maven mirror have already have all the dependencies there, the build time can be even faster.
+If the maven mirror have already have all the dependencies there, the build time can be even faster.
```
Comment (radtriste) [inline README.md]:
```suggestion
If the maven mirror already have all the dependencies there, the build time can be even faster.  
```

**Review by radtriste — APPROVED** [review]

**Comment by spolti** [comment]
@vaibhavjainwiz can you please rebase?

**Pull request merged**